### ISSUE TITLE
Vagrant package did not save my state. Please assist.

### ISSUE THREAD

**Issue by STashakkori** [mitchellh/vagrant#7266]
So yesterday I worked on my Vagrant deployed Virtual Box Lubuntu desktop VM with GUI enabled. After my day’s work I ran the command ‘vagrant package’. It closed my VM then complains that there is already a package.box file in that directory. Couldn’t it tell me that BEFORE it closed my VM session?? Or couldn’t it package it to a box with a different name for example package2.box?? Anyway, so then I ran ‘vagrant package --output package2.box’. After a really long time I discover the great news: that it did not save the state of the machine and that I lost all of my work for the day. Please tell me where I went wrong or where you went wrong so that I may avoid this annoyance in the future. Thank you.

### Vagrant version
1.8.1

### Host operating system
Windows 7

### Guest operating system
Lubuntu14

**Comment by sethvargo** [comment]
Hi @STashakkori 

Thank you for reaching out and opening an issue, and I'm sorry you experienced this issue. You're definitely correct that a possible improvement to `vagrant package` is to check if the box-file can be written to disk before starting the process. I've captured that [here](https://github.com/mitchellh/vagrant/issues/7351).

With respect to "saving the state of the machine". Vagrant does not maintain the machine's data - that is the virtualization layer's responsibility. Without more information, it's hard to say what "went wrong" because it's unclear what you mean by "state of the machine". Could you please elaborate?

**Comment by sethvargo** [comment]
Hey there,

I am going to close this due to lack of response. If this is still occurring, please open a _new issue_ and follow the provided issue template that appears when you click the "New Issue" button. This will help us in getting a reproduction and fix. Thanks! 😄
